migration_stock_picking_multiple_carrier/__openerp__.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Author: Rami Alwafaie
+#    Copyright 2015 initOS GmbH
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+{'name': 'Migration Multiple Carrier',
+ 'version': '1.0',
+ 'category': '',
+ 'description': """
+Migration - Multiple Carrier
```

Comment:
Extract the description to a README.rst file and use the template: https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst